<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rapid Classification AI Trainer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/UI_Icons.qrc">
    <normaloff>:/RCAIT.ico</normaloff>:/RCAIT.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
          border: 2px solid rgb(218,218,218);
          border-radius: 10px;
          margin-top: 6px;
          }
          QGroupBox::title {
          subcontrol-origin: margin;
          left: 12px;
          padding: 0px 3px 0px 3px;
          }

          QScrollBar:horizontal
          {
          height: 15px;
          margin: 3px 15px 3px 15px;
          border: 1px transparent #2A2929;
          border-radius: 4px;
          background-color: #FAFAFA; /* white bg */
          }

          QScrollBar::handle:horizontal
          {
        background-color: #C2C2C2;      /* grey */
        min-width: 5px;
        border-radius: 4px;
    }

    QScrollBar::add-line:horizontal
    {
        margin: 0px 3px 0px 3px;
        border-image: url(:/UISymbols/UI_Triangle_Right.svg);
        width: 10px;
        height: 10px;
        subcontrol-position: right;
        subcontrol-origin: margin;
    }

    QScrollBar::sub-line:horizontal
    {
        margin: 0px 3px 0px 3px;
        border-image: url(:/UISymbols/UI_Triangle_Left.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: left;
        subcontrol-origin: margin;
    }

    QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on
    {
        border-image: url(:/UISymbols/UI_Triangle_Right.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: right;
        subcontrol-origin: margin;
    }


    QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on
    {
        border-image: url(:/UISymbols/UI_Triangle_Left.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: left;
        subcontrol-origin: margin;
    }

    QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
    {
        background: none;
    }


    QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
    {
        background: none;
    }

    QScrollBar:vertical
    {
        background-color: #FAFAFA;  /* white */
        width: 15px;
        margin: 15px 3px 15px 3px;
        border: 1px transparent #2A2929;
        border-radius: 4px;
    }

    QScrollBar::handle:vertical
    {
        background-color: #C2C2C2;         /* grey */
        min-height: 5px;
        border-radius: 4px;
    }

    QScrollBar::sub-line:vertical
    {
        margin: 3px 0px 3px 0px;
        border-image: url(:/UISymbols/UI_Triangle_Top.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: top;
        subcontrol-origin: margin;
    }

    QScrollBar::add-line:vertical
    {
        margin: 3px 0px 3px 0px;
        border-image: url(:/UISymbols/UI_Triangle_Bottom.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
    }

    QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
    {
        border-image: url(:/UISymbols/UI_Triangle_Top.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: top;
        subcontrol-origin: margin;
    }

    QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on
    {
        border-image: url(:/UISymbols/UI_Triangle_Bottom.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
    }

    QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
    {
        background: none;
    }

    QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
    {
        background: none;
    }</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="CustomTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>55</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="StartWidget" name="tab_start">
       <attribute name="icon">
        <iconset resource="../Resources/UI_Icons.qrc">
         <normaloff>:/TabIcons/Filled/Tab_Start_Icon.svg</normaloff>:/TabIcons/Filled/Tab_Start_Icon.svg</iconset>
       </attribute>
       <attribute name="title">
        <string comment="tab_start_text">Home</string>
       </attribute>
      </widget>
      <widget class="ImportFilesWidget" name="tab_import">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="../Resources/UI_Icons.qrc">
         <normaloff>:/TabIcons/Filled/Tab_Crawler_Icon.svg</normaloff>:/TabIcons/Filled/Tab_Crawler_Icon.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Import Files</string>
       </attribute>
      </widget>
      <widget class="ImageInspectionWidget" name="tab_image_inspection">
       <attribute name="icon">
        <iconset resource="../Resources/UI_Icons.qrc">
         <normaloff>:/TabIcons/Filled/Tab_Input_Images_Icon.svg</normaloff>:/TabIcons/Filled/Tab_Input_Images_Icon.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Image Inspection</string>
       </attribute>
      </widget>
      <widget class="AITrainingWidget" name="tab_ai_training">
       <attribute name="icon">
        <iconset resource="../Resources/UI_Icons.qrc">
         <normaloff>:/TabIcons/Filled/Tab_Training_Images_Icon.svg</normaloff>:/TabIcons/Filled/Tab_Training_Images_Icon.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>AI Training</string>
       </attribute>
      </widget>
      <widget class="InputImagesWidget" name="tab_inputimages">
       <attribute name="icon">
        <iconset resource="../Resources/UI_Icons.qrc">
         <normaloff>:/TabIcons/Filled/filter_black_24dp.svg</normaloff>:/TabIcons/Filled/filter_black_24dp.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Input Images</string>
       </attribute>
      </widget>
      <widget class="ResultsWidget" name="tab_results">
       <attribute name="icon">
        <iconset resource="../Resources/UI_Icons.qrc">
         <normaloff>:/TabIcons/Filled/Tab_Results_Icon.svg</normaloff>:/TabIcons/Filled/Tab_Results_Icon.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Results</string>
       </attribute>
      </widget>
      <widget class="AutomationWidget" name="tab_automation">
       <attribute name="icon">
        <iconset resource="../Resources/UI_Icons.qrc">
         <normaloff>:/TabIcons/Filled/batch_prediction_black_24dp.svg</normaloff>:/TabIcons/Filled/batch_prediction_black_24dp.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Automation</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTabWidget</class>
   <extends>QTabWidget</extends>
   <header>customtabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InputImagesWidget</class>
   <extends>QWidget</extends>
   <header>inputimageswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultsWidget</class>
   <extends>QWidget</extends>
   <header>resultswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StartWidget</class>
   <extends>QWidget</extends>
   <header>startwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AITrainingWidget</class>
   <extends>QWidget</extends>
   <header>aitrainingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImportFilesWidget</class>
   <extends>QWidget</extends>
   <header>importfileswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AutomationWidget</class>
   <extends>QWidget</extends>
   <header>automationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageInspectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">imageinspectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/UI_Icons.qrc"/>
 </resources>
 <connections/>
</ui>
